pkg/col/coldata/batch.go

```diff
@@ -156,17 +156,17 @@ func (b *zeroBatch) Reset([]coltypes.T, int) {
 // MemBatch is an in-memory implementation of Batch.
 type MemBatch struct {
 	// length of batch or sel in tuples
-	n uint16
+	n uint64
 	// slice of columns in this batch.
 	b      []Vec
 	useSel bool
 	// if useSel is true, a selection vector from upstream. a selection vector is
 	// a list of selected column indexes in this memBatch's columns.
-	sel []uint16
+	sel []uint64
```

Comment:
Hmm, I remember that jordan ran some benchmarks with this change and performance decreased significantly

Reply:
I guess we should wait for Jordan with this PR.